worlds/wargroove2/Levels.py

```diff
@@ -0,0 +1,416 @@
+from typing import List
+
+from BaseClasses import Region, Entrance, MultiWorld
+from .Locations import location_table, Wargroove2Location
+from worlds.generic.Rules import set_rule
+
+region_names: [str] = ["North 1", "East 1", "South 1", "West 1",
+                       "North 2A", "North 2B", "North 2C",
+                       "East 2A", "East 2B", "East 2C",
+                       "South 2A", "South 2B", "South 2C",
+                       "West 2A", "West 2B", "West 2C",
+                       "North 3A", "North 3B", "North 3C",
+                       "East 3A", "East 3B", "East 3C",
+                       "South 3A", "South 3B", "South 3C",
+                       "West 3A", "West 3B", "West 3C"]
+FINAL_LEVEL_1 = "Northern Finale"
+FINAL_LEVEL_2 = "Eastern Finale"
+FINAL_LEVEL_3 = "Southern Finale"
+FINAL_LEVEL_4 = "Western Finale"
+
+LEVEL_COUNT = 28
+FINAL_LEVEL_COUNT = 4
+
+
+def set_region_exit_rules(region: Region, world: MultiWorld, player: int, locations: List[str], operator: str = "or"):
```

Comment:
Please do not do `world: MultiWorld`. Either do `multiworld: MultiWorld` or load the world class in.